### PR TITLE
Fixed for Python3

### PR DESCRIPTION
**Pull request by coreyhu**
Followed femtotrader's suggestions for fixing print statements for python3. Also updated README with instructions for using python3 -m http.server 8000 to start the webserver.

### CODE REVIEW

File: post3/runner.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 The main file that will evolve into our trading library
 """
-import v20
 from datetime import datetime, timedelta
+import v20
+
 
-OANDA_ACCESS_TOKEN = "YOUR_ACCESS_TOKEN"
-OANDA_ACCOUNT_ID = '11111' #put your access id here 
+OANDA_ACCESS_TOKEN = "01d4e6de4fc1246d2bfde02450871fe6-0ae159e87f2cc523a9116d9b535e85fa"
```
Comment (RaeesBhatti) [inline post3/runner.py]: Are you sure you want this here?

Comment (jonromero) [reply post3/runner.py]: Thanks for reviewing @elsteelbrain ! 

I should have used a .config file so people don't publish their keys online!

**Comment by jonromero** [comment]
Hey @coreywho thanks for the commits! Can you please remove your keys and do another pull request?

Thanks

**Comment by coreyhu** [comment]
Good catch. Here's the update!

**Comment by jonromero** [comment]
thanks!

**Pull request merged**